finagle-http/src/main/scala/com/twitter/finagle/http/codec/HttpServerDispatcher.scala

```diff
@@ -31,6 +31,11 @@ private[finagle] class HttpServerDispatcher(
     service.close()
   }
 
+  private def mayHaveContent(status: Status): Boolean = status match {
```

Comment:
I don't *think* we need this anymore, since we have the new Filter now.  What do you think?